content/attacks/posts/2019-11-27-Upbit.md

```diff
@@ -0,0 +1,29 @@
+---
+date: 2019-11-27
+custodians: Upbit
+categories: "Custodian"
+title: "Upbit Loses $49 Million in Cyberattack"
+---
+
+## Summary
+
+On November 27, 2019, Upbit, one of South Korea's largest cryptocurrency exchanges, fell victim to a major cyberattack, resulting in [the theft of approximately $49 million](https://www.coindesk.com/markets/2019/11/27/upbit-is-the-seventh-major-crypto-exchange-hack-of-2019/#:~:text=Upbit%20is%20the%20latest%20hacking,for%20at%20least%20two%20weeks.) worth of Ethereum (342,000 ETH). The attack specifically targeted Upbit's hot wallet, from which the funds were transferred to an unknown wallet address. To safeguard the remaining assets, Upbit promptly [moved them to cold storage](https://www.coindesk.com/markets/2020/01/14/upbit-exchange-resumes-ether-services-months-after-49m-hack/) as a safety measure. The exchange took immediate action, [replenishing the stolen funds with its own assets](https://upbit.com/service_center/notice?id=1085) and ensuring no impact on its customers.
```

Comment:
Please, shorten and clear source link

Reply:
Shortened the long link - removed a fragment identifier from it:
https://www.coindesk.com/markets/2019/11/27/upbit-is-the-seventh-major-crypto-exchange-hack-of-2019/#:~:text=Upbit%20is%20the%20latest%20hacking,for%20at%20least%20two%20weeks.